script/vcr_mcp/inspector.rb

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'json'
+require 'yaml'
+require 'base64'
+
+require_relative 'constants'
+require_relative 'cassette_parser'
+
+module VcrMcp
+  module Inspector
+    CASSETTE_ROOT = Constants::CASSETTE_ROOT
+
+    def self.find_cassette(query)
```

Comment:
The `find_cassette()` method in `script/vcr_mcp/inspector.rb:15-16` accepts user-controlled paths without validation, allowing an attacker to read any file on the system by passing absolute paths like `/etc/passwd` or relative paths like `../../config/database.yml`. The method immediately returns the path if `File.exist?(query)` is true, bypassing any cassette directory restrictions and exposing sensitive configuration files containing database credentials and API keys. 

A proper path sanitization pattern already exists in [`script/vcr_mcp/ssl_patcher.rb:182-188`](script/vcr_mcp/ssl_patcher.rb#L182-L188) that uses `File.expand_path()` and validates the resolved path starts with `VETS_API_ROOT`, which should be applied to `find_cassette()`.